### PR TITLE
Feature/#231 restaurant name in the browser

### LINKED ISSUE #231: [SEO] restaurant name in the browser

### PR DESCRIPTION
**Pull request by manudous**
Closed #231

### CODE REVIEW

File: menu-public-portal/src/pods/restaurant/restaurant.component.tsx [hunk 1]
```diff
@@ -139,78 +145,87 @@ export const RestaurantComponent: React.FC<Props> = (props) => {
...
-        <div className={classes.rowIndent}>
-          <Typography
-            variant="subtitle1"
-            component="h2"
-            className={classes.typographyHeader}
-          >
-            {phone}
+    <>
+      <Head>
+        <title>{name} - gastrocarta</title>
```
Comment (Nasdan) [inline menu-public-portal/src/pods/restaurant/restaurant.component.tsx:150]: Gastrocarta ?

File: menu-public-portal/src/pods/restaurant/restaurant.component.tsx [hunk 2]
```diff
@@ -139,78 +145,87 @@ export const RestaurantComponent: React.FC<Props> = (props) => {
...
-      <div className={classes.headerIndent}>
-        <div className={classes.rowIndent}>
-          <Typography
-            variant="subtitle1"
-            component="h2"
-            className={classes.typographyHeader}
-          >
-            {phone}
+    <>
+      <Head>
```
Comment (Nasdan) [inline menu-public-portal/src/pods/restaurant/restaurant.component.tsx:149]: Why don't use Heade in pages instead?

File: menu-public-portal/src/pods/restaurant/restaurant.component.tsx (hunk 1 above)
Comment (manudous) [reply menu-public-portal/src/pods/restaurant/restaurant.component.tsx]: done

File: menu-public-portal/src/pods/restaurant/restaurant.component.tsx (hunk 2 above)
Comment (manudous) [reply menu-public-portal/src/pods/restaurant/restaurant.component.tsx]: done

**Pull request merged**